app/Controllers/CtrlCertificate.php

```diff
@@ -31,22 +34,35 @@ public function viewCertificateEdit($id)
 
     public function createCertificate()
     {
-        $isCreated = true;
-        if ($isCreated) {
+        try {
+            $validateCertificate = new CertificateValidation();
+            $certificateData     = $this->request->getPost();
+            if (! $validateCertificate->validateInputs($certificateData)) {
+                throw new InvalidInputException($validateCertificate->getErrors());
+            }
+
+            $certificateData['certificatePreviewId'] = '4fe307c224811d55048d72b5696895eb';
+            $certificateData['certificatefileId']    = '4fe307c224811d55048d72b5696895eb';
```

Comment:
That hash code represents the identifier of a temporary folder, so add a folder with that identifier and its associated file or use the existing one, for example: `3ea307c224811d55048d72b5696895eb`, otherwise it generates an error. 
```suggestion
            $certificateData['certificatePreviewId'] = '3ea307c224811d55048d72b5696895eb';
            $certificateData['certificatefileId']    = '3ea307c224811d55048d72b5696895eb';
```